src/csm_testing/tests/sat_functional/bootprep/test_bootprep.py

```diff
@@ -712,8 +726,11 @@ def test_session_template(self):
             self.items_to_delete['session_templates'].append(image['name'])
 
     def test_configs_images_and_session_templates(self):
-        """Test creating a bos session template"""
-        result = self.run_bootprep('configs-images-and-session-templates.yaml', '--format json')
+        """Test creating, skipping and overwriting configurations, images and session templates"""
+        bootprep_options = '--format json '
+        result = self.run_bootprep('configs-images-and-session-templates.yaml', bootprep_options)
```

Comment:
It would be possible to reduce the time taken by this test if we created the CFS configuration, IMS image, and BOS session template using `cray` commands first. It should be possible to create an IMS image record that doesn't actually have any artifacts in it, so there would be no need to actually build or configure a real image.